delivery_recipe/digital_service_standards.md

```diff
@@ -2,10 +2,17 @@
 
 As part of driving quality through our delivery process we incorporate the following Digital Service Standards:
```

Comment:
No, it's not referring to the gov Service Design Standards. These are just the collection of standards we work to, for all clients, whether public or private sector. They aim for compatibility with the government standards, but also have to include things like our ISO9001/27001 processes, and ways that we've developed to work in.